### PR TITLE
feat: add outreachy blog in otel blogs

### PR DESCRIPTION
**Pull request by sakshi-1505**
Outreachy blog inspired from https://sakshipatle.substack.com/p/outreachy-how-when-and-why

### CODE REVIEW

**Comment by sakshi-1505** [comment]
@svrnm I will make draft:false once the PR gets approved

File: content/en/blog/2023/outreachy-how-why-sakshi.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+date: 2023-12-24 # Put the current date, we will keep the date updated until your PR is merged
+author:
+  >- # If you have only one author, then add the single name on this line in quotes.
+  [Sakshi Patle](https://github.com/sakshi-1505) (OpenTelemetry),
+draft: true # TODO: remove this line once your post is ready to be published
+# canonical_url: http://somewhere.else/ # TODO: if this blog post has been posted somewhere else already, uncomment & provide the canonical URL here.
+---
+
+Hello world! I’m Sakshi Patle, a final-year Computer Science and Engineering
+student at RGPV in the heart of India. My journey into the tech universe has
```
Comment (svrnm) [inline content/en/blog/2023/outreachy-how-why-sakshi.md:13]:
```suggestion
student at RGPV in the heart of India. My journey into the tech universe has
```

What does RGPV stand for? Write it out and then put the abbreviation in (...)

File: content/en/blog/2023/outreachy-how-why-sakshi.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+---
+title: Outreachy - How, When & Why?
+linkTitle: Introduction to Outreachy 2023 # Mandatory, make sure that your short title.
+date: 2023-12-24 # Put the current date, we will keep the date updated until your PR is merged
+author:
+  >- # If you have only one author, then add the single name on this line in quotes.
+  [Sakshi Patle](https://github.com/sakshi-1505) (OpenTelemetry),
+draft: true # TODO: remove this line once your post is ready to be published
+# canonical_url: http://somewhere.else/ # TODO: if this blog post has been posted somewhere else already, uncomment & provide the canonical URL here.
```
Comment (svrnm) [inline content/en/blog/2023/outreachy-how-why-sakshi.md:9]: you can add a pointer to your medium post here.

File: content/en/blog/2023/outreachy-how-why-sakshi.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
+---
+title: Outreachy - How, When & Why?
+linkTitle: Introduction to Outreachy 2023 # Mandatory, make sure that your short title.
+date: 2023-12-24 # Put the current date, we will keep the date updated until your PR is merged
+author:
+  >- # If you have only one author, then add the single name on this line in quotes.
+  [Sakshi Patle](https://github.com/sakshi-1505) (OpenTelemetry),
+draft: true # TODO: remove this line once your post is ready to be published
```
Comment (svrnm) [inline content/en/blog/2023/outreachy-how-why-sakshi.md:8]:
```suggestion
```

File: content/en/blog/2023/outreachy-how-why-sakshi.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
+---
+title: Outreachy - How, When & Why?
+linkTitle: Introduction to Outreachy 2023 # Mandatory, make sure that your short title.
```
Comment (svrnm) [inline content/en/blog/2023/outreachy-how-why-sakshi.md:3]:
```suggestion
linkTitle: Introduction to Outreachy 2023 # Mandatory, make sure that your short title.
```

The linkTitle should be short and only a few words, like "Outreachy Internship" or "Outreachy Report", etc.

File: content/en/blog/2023/outreachy-how-why-sakshi.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+technology. During my academic tenure, I grappled with concepts, whether it was
+establishing CI/CD pipelines or developing Golang projects like a banking app
+with REST APIs, and gRPC services for my college portal. These challenges tested
+my resolve but also reinforced my belief in perseverance. It's the commitment to
+keep going, to learn and to improve, that ultimately transforms setbacks into
+successes.
+
+## The Motivation Behind Outreachy
+
+Outreachy for me was not just a program; it represented the intersection of my
```
Comment (svrnm) [inline content/en/blog/2023/outreachy-how-why-sakshi.md:48]:
```suggestion
[Outreachy](https://www.outreachy.org/) for me was not just a program; it represented the intersection of my
```

File: content/en/blog/2023/outreachy-how-why-sakshi.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+the very tenets of Outreachy—diversity, inclusivity, and empowerment.
+
+Outreachy offers a nurturing ground to enhance skill sets, build confidence, and
+foster community engagement, which aligns perfectly with my aspiration to be a
+valuable contributor to the OTel community while embodying my core values.
+
+## Why OpenTelemetry?
+
+OpenTelemetry (OTel) stood out to me as a space where security and scalability
+are not just focused on but are thriving. As an intern with the SIG Security
```
Comment (svrnm) [inline content/en/blog/2023/outreachy-how-why-sakshi.md:65]:
```suggestion
are not just focused on but are thriving. As an intern with the [SIG Security](https://github.com/open-telemetry/sig-security)
```

File: content/en/blog/2023/outreachy-how-why-sakshi.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+team, I’ll be championing the implementation of organization-wide repository
+governance policies and checks. This isn’t just about enforcing standards; it’s
```
Comment (svrnm) [inline content/en/blog/2023/outreachy-how-why-sakshi.md:67]:
```suggestion
team, I’ll be championing the implementation of organization-wide repository
governance policies and checks. This isn’t just about enforcing standards; it’s
```

is there a GitHub issue that you can link from here? Do you want to mention AllStar somewhere?

File: content/en/blog/2023/outreachy-how-why-sakshi.md [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+title: Outreachy - How, When & Why?
+linkTitle: Introduction to Outreachy 2023 # Mandatory, make sure that your short title.
+date: 2023-12-24 # Put the current date, we will keep the date updated until your PR is merged
+author:
+  >- # If you have only one author, then add the single name on this line in quotes.
+  [Sakshi Patle](https://github.com/sakshi-1505) (OpenTelemetry),
+draft: true # TODO: remove this line once your post is ready to be published
+# canonical_url: http://somewhere.else/ # TODO: if this blog post has been posted somewhere else already, uncomment & provide the canonical URL here.
+---
+
```
Comment (svrnm) [inline content/en/blog/2023/outreachy-how-why-sakshi.md:11]: TODO for me: As discussed I will add a leading paragraph

**Review by svrnm — CHANGES_REQUESTED** [review]
Adding the slack thread for context: https://cloud-native.slack.com/archives/C05A85QC281/p1702464743038639?thread_ts=1702464110.897059&cid=C05A85QC281

Thank you @sakshi-1505, I did a first run of reviews, please take a look

**Comment by svrnm** [comment]
@sakshi-1505 I added some words as introduction and did some editing of the content, please take a look if everything is correct (e.g. I assumed that RGPV is Rajiv Gandhi Proudyogiki Vishwavidyalaya, please correct this if I am incorrect)

**Review by svrnm — APPROVED** [review]

**Comment by svrnm** [comment]
Preview: https://deploy-preview-3710--opentelemetry.netlify.app/blog/2024/outreachy-how-why-sakshi/

@jpkrohling can you take a quick look as well, thx!

**Review by jpkrohling — APPROVED** [review]

File: content/en/blog/2024/outreachy-how-why-sakshi.md [hunk 9]
```diff
@@ -0,0 +1,83 @@
+---
+title: Outreachy - How, When & Why?
+linkTitle: Outreachy Internship
+date: 2024-01-12
```
Comment (svrnm) [inline content/en/blog/2024/outreachy-how-why-sakshi.md:4]:
```suggestion
date: 2024-01-17
```

**Pull request merged**